tests/auth_tests/test_views.py

```diff
@@ -551,6 +551,23 @@ def test_security_check(self, password='password'):
             self.assertEqual(response.status_code, 302)
             self.assertIn(good_url, response.url, "%s should be allowed" % good_url)
 
+    def test_security_check_https(self, password='password'):
```

Comment:
No need for a `password` parameter. I'll remove that current pattern... not sure how it got started.